nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
While rubber ducking with @SimonCropp I came to the conclusion that we don't need a version table. The sequence of disposing the bus and then disposing the container is a valid sequence under all circumstances. If you pass in a container into the bus configuration you have to create the container first. All those containers are disposable. when you create the container the code doing it becomes the composition root of the container and also the bus. Leaving aside the implementation details of UnicastBus the valid sequence of disposing is

1. Dispose the bus by calling IBus.Dispose().
2. Dispose the external container with container.Dispose()

in all scenarios independent from the backported changes. So the documentation in that sense holds up and we don't need a version table. @indualagarsamy ping me when you are online